src/App.js

```diff
@@ -250,9 +251,17 @@ class App extends Component {
     render = () => {
         const {schema, uiSchema, formData, hasError, hasSuccess, submissionStatus} = this.state;
         const onSubmit = ({formData}) => this.submitForm(formData);
+	
+        // this.props.children come through as a DocumentFragment, 
+        // and cannot be added directly as a React element.
+        let tempElem = document.createElement("div");
+        tempElem.appendChild(this.props.children);
+        let childrenHtmlStr = tempElem.innerHTML;
+        console.log("this.props.children is: ["+ childrenHtmlStr + "]");
```

Comment:
```suggestion
```

Reply:
Thx for the catch - once we're ready to merge, I'll remove it - that log statement is helping for now with the issue of the component losing it's state/props.